src/sql/base/parts/editableDropdown/browser/dropdown.ts

```diff
@@ -241,6 +242,13 @@ export class Dropdown extends Disposable {
 							this._treeContainer.remove();
 						}
 					};
+				},
+				onDOMEvent: e => {
+					if (!DOM.isAncestor((<HTMLElement>e.target), this._el) && !DOM.isAncestor((<HTMLElement>e.target), this._treeContainer)) {
```

Comment:
We shouldn't need this anymore, I removed this because now the onBlur code handles tearing everything down when the widget loses focus

Reply:
yeah, you are right, we don't need this.